docker-bake.hcl

```diff
@@ -52,7 +56,7 @@ function "patch" {
 
 function "tag" {
     params = [tag]
-    result = ["${REPO}:${tag}"]
+    result = ["${REPO}:${tag}", "${GHCR_REPO}:${tag}"]
```

Comment:
I think this defeats the purpose of the the `REPO` variable. If a user that is not use want's to tag a build with their own repository, specifying `REPO` will still also tag GH. Maybe make the two repos fixed and default and allow users to supply a list of repositories instead?

Reply:
Sounds good.